src/conn/conn_sweep.c

```diff
@@ -50,19 +50,43 @@ __sweep_mark(WT_SESSION_IMPL *session, uint64_t now)
 }
 
 /*
- * __sweep_expire_one --
- *     Mark a single handle dead.
+ * __sweep_close_dhandle_locked --
+ *     Close write-locked dhandle.
  */
 static int
-__sweep_expire_one(WT_SESSION_IMPL *session)
+__sweep_close_dhandle_locked(WT_SESSION_IMPL *session)
 {
     WT_BTREE *btree;
     WT_DATA_HANDLE *dhandle;
-    WT_DECL_RET;
 
     dhandle = session->dhandle;
     btree = WT_DHANDLE_BTREE(dhandle) ? dhandle->handle : NULL;
 
+    /* This method expects dhandle write lock. */
+    WT_ASSERT(session, (dhandle->lock_flags | WT_DHANDLE_LOCK_WRITE) == WT_DHANDLE_LOCK_WRITE);
```

Comment:
This could be:
FLD_ISSET(dhandle->lock_flags, WT_DHANDLE_LOCK_WRITE)

Reply:
Done, thanks!